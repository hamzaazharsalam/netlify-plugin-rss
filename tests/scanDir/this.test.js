const fs = require('fs');
const path = require('path');

// resolve all files
const PUBLISH_DIR = path.join(__dirname, 'publishDir');

// debug
// console.log({ publishDirPath, manifest });

// actual test
const pluginCore = require('../../pluginCore.js');
test('scanDir works', async () => {
  const allHtml = await pluginCore.scanDir({
    dirToScan: '/',
    PUBLISH_DIR,
    debugMode: false, // to future readers - turn this true to make pluginCore log out more stuff
    testMode: true
  });
  expect(allHtml).toMatchSnapshot();
});
