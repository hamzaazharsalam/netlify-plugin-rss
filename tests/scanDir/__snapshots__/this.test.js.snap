// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`scanDir works 1`] = `
Array [
  "subdir/blog3.html",
  "blogs/blog3/blog2.html",
]
`;
